src/scheduler/data_types.h

```diff
@@ -824,6 +823,7 @@ struct resource_resv
 	job_info *job;			/* pointer to job specific structure */
 	resv_info *resv;		/* pointer to reservation specific structure */
 
+	char *svr_inst_id;	/* Server instance id of the job/reservation */
```

Comment:
Align the comment with the others.

Reply:
they aren't really consistent, some use 2 tabs, some use 3, but ill use 2 instead of 1 to try to be consistent with some